src/routes/formsg/formsgSiteLaunch.ts

```diff
@@ -26,12 +26,12 @@ import { DigResponse, DigType } from "@root/types/dig"
 import UsersService from "@services/identity/UsersService"
 import InfraService from "@services/infra/InfraService"
 
-const { SITE_LAUNCH_FORM_KEY } = process.env
+const SITE_LAUNCH_FORM_KEY = config.get("formSg.siteLaunchFormKey")
```

Comment:
not sure why this wasnt migrated.. considering that we dont use env vars anymore I am abit surprised how this works

Reply:
Possibly we might not have had any site launches since the node 18 swapover? On the node 16 env we didn't delete our existing env vars, so it's possible that it was still reading from there

Reply:
it is a safe change, since ssm populates our env. we are just changing the place where it reads here. resolving this ya